### DISCUSSION TITLE
RSS feed for GitHub issues?

### DISCUSSION THREAD

**Discussion by adm2k** [FreshRSS/FreshRSS#5378] [Q&A]
Is there a feed I can add to Fress RSS for following Fresh RSS Github issues?

https://github.com/FreshRSS/FreshRSS/issues

This fails, https://openrss.org/github.com/FreshRSS/FreshRSS/issues

Thank you!

**Comment by Alkarex** [comment]
https://openrss.org/github.com/FreshRSS/FreshRSS/issues works for me if you change the user-agent and increase the timeout in the advanced settings when adding the feed.

Otherwise, according to https://github.com/orgs/community/discussions/7871 this is not directly supported by GitHub.

**Comment by adm2k** [comment]
Changed user-agent to Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

and timeout to 5

validating the feed still errors:
This feed does not validate.

[line 1](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fopenrss.org%2Fgithub.com%2FFreshRSS%2FFreshRSS%2Fissues#l1), column 311: Invalid email address: adm2k (5 occurrences) [[help](https://validator.w3.org/feed/docs/error/InvalidContact.html)]

... ![CDATA[RSS feed]]></title><author>adm2k</author><link><![CDATA[https:// ...
                                             ^
In addition, interoperability with the widest range of feed readers could be improved by implementing the following recommendations.

[line 1](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fopenrss.org%2Fgithub.com%2FFreshRSS%2FFreshRSS%2Fissues#l1), column 0: Invalid HTML: Unexpected start tag td. Ignored. (7 occurrences) [[help](https://validator.w3.org/feed/docs/warning/NotHtml.html)]

<?xml version="1.0"?><?xml-stylesheet type="text/xsl" href="/github.com/Fres ...

**Reply by Alkarex** [reply]
5 seconds is much too low. Try 60

**Reply by adm2k** [reply]
Changed to 60 seconds and getting the same validation errors.

also, Just checked and my version shows as 1.21.1-dev How did I get on a "dev" version? How can I get a "stable version"

**Comment by adm2k** [comment]
Just checked and my version shows as 1.21.1-dev How did I get on a "dev" version? How can I get a "stable version"
